tools/depends/configure.ac

```diff
@@ -687,16 +683,12 @@ AC_SUBST(ffmpeg_options)
 AC_OUTPUT
 
 if test "$platform_os" == "darwin_embedded"; then
-  if test "$use_platform" = "tvos"; then
-    simulator_sdk_path=[`$use_xcodebuild -version -sdk appletvsimulator$use_sdk  | grep ^Path | awk '{ print $2}'`]
```

Comment:
Why are you removing this?

Reply:
We don't actually support building for tvos simulator. If someone does the legwork to introduce it, then we can re add then